src/common/types/commands/account_objects.ts

```diff
@@ -44,7 +49,16 @@ export interface AccountObjectsResponse {
   account: string,
 
   // Array of objects owned by this account.
-  account_objects: CheckLedgerEntry | object,
+  // from the getAccountObjects section of the dev center
+  account_objects: Array<
```

Comment:
good find. If we find multiple uses for this Union Type, we can refactor it into it's own type as well